dashboard/config/scripts/time4cs-control-unit-1.script

```diff
@@ -1,7 +1,6 @@
 login_required true
 hideable_stages true
 stage_extras_available true
-pilot_experiment 'time4cs-control'
```

Comment:
we had to remove pilot_experiment because this bug prevents people in the editor_experiment from editing when there is also a pilot experiment: [LP-1080](https://codedotorg.atlassian.net/browse/LP-1080)